common/tools/dev-tool/src/commands/run/index.ts

```diff
@@ -6,6 +6,8 @@ import { subCommand, makeCommandInfo } from "../../framework/command";
 export const commandInfo = makeCommandInfo("run", "run scripts such as test:node");
 
 export default subCommand(commandInfo, {
+  "test:node-tsx-ts": () => import("./testNodeTsxTS"),
```

Comment:
Not blocking but we will now have 6 commands for running tests 😨 Is the plan for these to eventually replace `test:node-ts-input` and `test:node-js-input` with the tsx variants? That's what I would expect but just checking!

And, we should decide what that means for vitest, and if that's still a direction we want to go...

Reply:
The plan will be to remove these old ones once we have moved everyone to `tsx` for now, and then will move everyone to vitest.  It's a couple step process here.